net2cog/utilities.py

```diff
@@ -9,8 +9,8 @@
 from logging import Logger
 import xarray as xr
 
-X_COORDINATE = ("lon", "longitude", "x", "x-dim")
-Y_COORDINATE = ("lat", "latitude", "y", "y-dim")
+X_COORDINATE = ("lon", "longitude", "x", "x-dim", "XDim")
+Y_COORDINATE = ("lat", "latitude", "y", "y-dim", "YDim")
```

Comment:
I guess this begs two questions:

1) Are these lists now exhaustive? (Probably not)
2) Is there a programmatic and more reliable way to determine dimension values from metadata attributes, instead of relying on dimension names?

These questions are probably both out-of-scope for this PR, but it's easy to see needing to keep coming back here each time a new mission wants to be processed by net2cog. It would be nice to head that off at some point.

Reply:
Still out of scope, but I'm thinking about the problem a bit more now...

I wonder if the answer to question 2 is that the check against `standard_name` and `units` should come first, and checking against known dimension names comes second. That might feel like better use of CF-Convention compliant metadata before resorting to variable names that don't have as stringent conditions.